src/bp/core/services/ghost/service.ts

```diff
@@ -202,25 +215,26 @@ export class GhostService {
     return scopedGhost
   }
 
-  public async exportArchive(botIds: string[]): Promise<Buffer> {
+  public async exportArchive(): Promise<Buffer> {
     const tmpDir = tmp.dirSync({ unsafeCleanup: true })
-    const files: string[] = []
+    const ignoredFiles = ['models/**', '**/*.js.map']
```

Comment:
you could extract this as a "top level" constant and use it in `getDirectoryFullPaths` as well. this way, if for whatever reason it changes it would be easier.

Reply:
I thought of adding it on the top, but they have a slight variation. When exporting the archive, you want the revisions.json file downloaded locally.

But when you compare the list of all files for added/delete, you want to ignore the revisions file because it's generated and edited by the archive generation process